tests/k8s/cluster.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+set -o verbose
+
+# based on
+# https://github.com/kubernetes/minikube#linux-continuous-integration-without-vm-support
+
+function k8s::install_minikube {
+    local minikube_version="v1.25.2"
+    sudo apt-get update
+    sudo apt-get install -y conntrack
+    curl -Lo minikube https://storage.googleapis.com/minikube/releases/${minikube_version}/minikube-linux-amd64
+    chmod +x minikube
+    sudo mv minikube /usr/local/bin/
+    sudo -E minikube config set WantReportErrorPrompt false
+    sudo -E minikube config set WantNoneDriverWarning false
+}
+
+function k8s::start {
+    export KUBECONFIG=$HOME/.kube/config
+    mkdir -p $(dirname $KUBECONFIG)
+    touch $KUBECONFIG
+
+    export MINIKUBE_WANTUPDATENOTIFICATION=false
+    export MINIKUBE_WANTREPORTERRORPROMPT=false
+    export MINIKUBE_HOME=$HOME
+    export CHANGE_MINIKUBE_NONE_USER=true
+
+    sudo -E minikube start \
+        --vm-driver=none \
+        --install-addons=true \
+        --addons=registry \
```

Comment:
is registry needed?

Reply:
@zubenkoivan yeah we need because we pull an admission controller lib from the GCR

Reply:
but this plugin install private registry inside minikube, we don't need it to pull images from GCR